en/release-notes/upgrade-guide-v3.0.md

```diff
@@ -54,56 +48,166 @@ Please get the `UITransform` component on the node first, and then use the corre
 node.getComponent(UITransform).setContentSize(size);
 ```
 
-#### loader
-
-The loader API is consistent with v2.4, please refer to the [v2.4 Resource Management Module Upgrade Guide](https://docs.cocos.com/creator/manual/en/release-notes/asset-manager-upgrade-guide.html).
-
 ### Editor upgrade
 
 #### Build Panel
 
-The builds of all platforms in __Cocos Creator 3.0 Preview__ are built-in plug-ins, so the build panel is also different from __Creator v2.4__. The unique build options of each platform will be placed in a foldable section control separately.
+The builds of all platforms in __Cocos Creator 3.0__ are built-in plug-ins, so the __Build__ panel is also different from __Creator v2.4__. The unique build options of each platform will be placed in a foldable section control separately.
 
-![image](https://user-images.githubusercontent.com/1503156/100602713-3d56ff80-333f-11eb-8280-d58e262ccc2b.png)
+![image](build-panel.png)
 
 After clicking the build button, it will jump to the **Build Tasks** panel, where all built platforms will be displayed. You can modify the build options of the built project in this panel and then rebuild, view the build log, open the project directory, etc. If you need to compile for other platforms, click the **New Build Task** button at the top left of the panel.
 
-![image](https://user-images.githubusercontent.com/1503156/100602806-5cee2800-333f-11eb-8dfe-4ba7e8e9283a.png)
+![image](build.png)
 
-In addition, it supports the construction of multi-module results with file separation during construction, which facilitates concurrent loading of multiple modules and dynamic loading of modules, and the WeChat engine plug-in supports the selection of different physical engine backends. The `settings.js` generated after the build is also changed to `settings.json` and placed under `res`, allowing it to be used as a resource upload server.
+In addition, it supports the construction of multi-module results with file separation during construction, which facilitates concurrent loading of multiple modules and dynamic loading of modules, and the WeChat engine plug-in supports the selection of different physical engine backends. The `settings.js` generated after the build is also changed to `settings.json` and placed under `src` directory, allowing it to be used as a resource upload server.
 
-#### Resource thumbnail panel
+#### Asset Preview Panel
 
-Select the resource in the resource manager to display resource thumbnails in the resource thumbnail panel for easy viewing:
+Select the resource in the __Asset__ panel to display resource thumbnails in the __Asset Preview__ panel. If select the folder where the resource is located, the thumbnails of all resources under the folder can be displayed for easy viewing.
```

Comment:
If select the folder where the resource is located, the thumbnails of all resources under the folder can be displayed for easy viewing.
-->
If the folder where the resource is located is selected, the thumbnails of all resources under the folder can be displayed for easy viewing.